python-package/lets_plot/plot/label.py

```diff
@@ -25,102 +25,96 @@ def ggtitle(label):
 
     Returns
     --------
+    `FeatureSpec`
         Plot title specification.
 
     Note
-    -----
-        Changes plot title.
+    ----
+    Changes plot title.
 
     Examples
-    ---------
-    >>> import numpy as np
-    >>> import pandas as pd
-    >>> from scipy.stats import multivariate_normal
-    >>> N = 100
-    >>> M = 3
-    >>> mean = np.zeros(M)
-    >>> cov = np.eye(M)
-    >>> X = multivariate_normal.rvs(mean, cov, N)
-    >>> X = X.astype(int) # comment this line to make variables continuous back
-    >>> dat = pd.DataFrame(X)
-    >>> dat = pd.melt(dat)
-    >>> ggplot(dat, aes(x='value', group='variable', fill='variable')) +
-    ... geom_bar(stat='bin', position=position_dodge(width=5.0), width=10, alpha=0.8) +
-    ... ggtitle('Plot title') + xlab('x axis label') + ylab('y axis label')
+    --------
+    .. jupyter-execute::
+        :linenos:
+        :emphasize-lines: 7
+        
+        from lets_plot import *
+        LetsPlot.setup_html()
+        data = {}
+        data['x'] = [x for x in range(10)]
+        data['y'] = [y for y in range(10)]
+        ggplot(data=data) + geom_point(aes(x='x', y='y', size='y')) + \\
+            ggtitle(label='New plot title')
```

Comment:
It can be written shorter:
```python
from lets_plot import *
LetsPlot.setup_html()
data = {'x': list(range(10)), 'y': list(range(10))}
ggplot(data, aes('x', 'y')) + geom_point(aes(size='y')) + \
    ggtitle('New Plot Title')
```

And even:
```python
from lets_plot import *
LetsPlot.setup_html()
ggplot() + geom_point(x=0, y=0) + \
    ggtitle(label='New Plot Title')
```
Here I removed all that doesn't matter.

You can don't follow all my suggestions, but at least simplify generating of the data.